x/ccv/child/types/keys.go

```diff
@@ -51,11 +51,15 @@ const (
 
 	// CrossChainValidatorPrefix is the key prefix that will store cross-chain validators by consensus address
 	CrossChainValidatorPrefix = "crosschainvalidator"
+
+	// ConsumerRedistributeName the root string for the consumer-redistribution account address
+	ConsumerRedistributeName = "cons_redistribute"
```

Comment:
Why isn't this a param? Just curious, not saying it should be

Reply:
Because it doesn't need to be a param, really any further logic should be hardcoded to this use the consumer redistribution address, for instant a "send to the developers multisig" could easily be an automatic transaction from this address. This is similar to the fees address which is hardcoded for the application. Closing issue, feel free to reopen if you want to discuss more